log/sequencer.go

```diff
@@ -66,20 +67,41 @@ var (
 	QuotaIncreaseFactor = 1.1
 )
 
+type stringSet map[string]bool
```

Comment:
I know it's only "temporary", but might be worth a comment on this - it's mostly a string set except when there's a magical string present in the set in which case the rest of the set is dropped.

Just something to reduce surprise if someone inadvertently tried to use this stringSet for something else in here :)